CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # nemo-screenshot changelog
 
+## v2.2.1
```

Comment:
please change header name to UNRELEASED (publisher will manage final changelog version